packages/runner/src/boundary.ts

```diff
@@ -0,0 +1,19 @@
+import { randomBytes } from "crypto";
+import { promisify } from "util";
+
+export class Boundary {
+  public readonly error: string;
+  public readonly start: string;
+  public readonly end: string;
+
+  public static async create(): Promise<Boundary> {
+    const buf = await promisify(randomBytes)(16);
```

Comment:
Not sure how long randomBytes takes but I think I already made a synchronous random string generator somewhere in this repo and we don't need it to be cryptographically secure

Reply:
> we don't need it to be cryptographically secure

Are you sure? Thinking about alf.nu/returnTrue's `random4` example 😆